cypress/integration/ui/edit_dashboard/edit_dashboard.js

```diff
@@ -95,6 +95,16 @@ Given('I open existing dashboard', () => {
     cy.get(dashboardChipSel, EXTENDED_TIMEOUT)
         .contains(TEST_DASHBOARD_TITLE)
         .click()
+
+    cy.location().should(loc => {
+        const currentRoute = getRouteFromHash(loc.hash)
+
+        expect(nonViewRoutes).not.to.include(currentRoute)
```

Comment:
This works, but my spontaneous thought was to check that it's a valid uid (i.e. valid format, not valid in the db).

Reply:
Actually, maybe that's what you're doing on the line below though? 🤔

Reply:
Yepp